packages/round-manager/src/features/round/ApplicationDirectPayout.tsx

```diff
@@ -0,0 +1,466 @@
+                </p>
+                <p className="text-sm">
+                  {(round.protocolFeePercentage || 0) * 100} %
+                </p>
+              </div>
+              <div className="flex flex-row justify-start mt-4">
+                <p className="flex flex-row text-sm w-1/3">
+                  Round fee:
+                  <span>
+                    <InformationCircleIcon
+                      data-tip
+                      data-background-color="#0E0333"
+                      data-for="round-fee-tooltip"
+                      className="mt-1 ml-1 text-gray-900 w-3 h-3"
+                      data-testid={"round-fee-tooltip"}
+                    />
+                    <ReactTooltip
+                      id="round-fee-tooltip"
+                      place="bottom"
+                      type="dark"
+                      effect="solid"
+                    >
+                      <p className="text-xs">
+                        The round fees are any additional charges
+                        <br />
+                        for services used to run your round. These
+                        <br />
+                        can be software services (i.e. this user interface)
+                        <br />
+                        or other specialty tools. If enabled, they are
+                        <br />
+                        calculated as a percentage of your funding pool.
+                      </p>
+                    </ReactTooltip>
+                  </span>
+                </p>
+                <p className="text-sm">
+                  {(round.roundFeePercentage || 0) * 100}%
+                </p>
+              </div>
+            </div>
+            <p className="rounded-md bg-violet-100 py-4 px-4 text-violet-400 flex mt-2 text-sm">
+              <InformationCircleIcon className="w-4 h-4 mr-2 mt-0.5" />
+              <span className="font-medium">
+                <strong>Important:</strong>
+                <p>
+                  Make sure the vault address has a balance of at least{" "}
+                  {getAmountWithFee()}
+                  DAI.
+                </p>
+                {allInputs.address &&
+                  address.toLowerCase() !== allInputs.address.toLowerCase() && (
+                    <p>
+                      Make sure the vault address has allowed the payout
+                      contract with address {round.payoutStrategy.id} to spend{" "}
+                      {getAmountWithFee()} DAI.
+                    </p>
+                  )}
+              </span>
+            </p>
+          </div>
+          <div className="px-6 align-middle py-3.5 bg-[#F3F3F5]">
+            <Button
+              className="float-right"
+              type="submit"
+              data-testid="trigger-payment"
+            >
+              Payout
+            </Button>
+          </div>
+        </form>
+      </section>
+      <ProgressModal
+        isOpen={openPayoutProgressModal}
```

Comment:
`openPayoutProgressModel` is phrased as an action, implying a function, whereas it's a boolean. `isPayoutProgressModelOpen` would carry lower risk of the reader misunderstanding.